spec/graphql/upgrader/member_spec.rb

```diff
@@ -276,21 +278,22 @@ class UserInterface < BaseInterface
         field :example_connection, -> { ExampleConnectionType }, null: true, connection: true
           method: :example_connections
       }
-      assert_equal upgrade(old), new
-    end
-  end
 
-  describe 'field with description inside block' do
-    it 'moves the description as an argument' do
-      old = %{
-        field :_id, !types.ID, property: :id do
-          description 'The primary key from the database'
-        end
-      }
-      new = %{
-        field :_id, ID, 'The primary key from the database', method: :id, null: false
-      }
       assert_equal upgrade(old), new
     end
   end
+
+  # describe 'field with description inside block' do
+  #   it 'moves the description as an argument' do
+  #     old = %{
+  #       field :_id, !types.ID, property: :id do
+  #         description 'The primary key from the database'
```

Comment:
IRL I told you this field is no more, but I think I was mistaken. Looking at the docs it's still supported, did you try to fix something with this spec?

Reply:
:thinking: I'm pretty sure it broke, but presumably the docs aren't lying. Let's take a closer look asap,